src/components/theme/light-theme.css

```diff
@@ -1,25 +1,42 @@
 /* Default and primary button colors differ within modals */
-.light-theme :global(.ant-btn-default) {
-    background-color: var(--light-button-default-background);
-    border-color: var(--light-theme-modal-btn-default-color);
-    color: var(--light-button-default-color);
+
+/* primary button */
+.light-theme :global(.primary-button),
+modal :global(.primary-button):focus {
+    border: 1px solid var(--light-theme-button-primary-bg);
+    border-radius: 3px;
+    background-color: var(--light-theme-button-primary-bg);
+    color: var(--light-theme-button-primary-text);
+}
+
+.light-theme :global(.primary-button):hover {
+    color: var(--light-theme-button-primary-text-hover);
+    border: 1px solid var(--light-theme-button-primary-border-hover);
 }
 
-.light-theme :global(.ant-btn-primary) {
-    background-color: var(--light-theme-btn-primary-bg);
-    border: var(--light-button-primary-border);
-    color: var(--light-theme-btn-primary-color);
+.light-theme :global(.primary-button):disabled {
+    border: 1px solid var(--light-theme-button-primary-text-disabled);
+    color: var(--light-theme-button-primary-text-disabled);
+    background-color: var(--light-theme-button-primary-bg-disabled);
 }
 
-.light-theme :global(.ant-btn-primary[disabled]) {
-    background: var(--light-button-disabled-background);
-    border: var(--light-button-disabled-border);
-    color: var(--light-theme-btn-primary-color);
+/* secondary button */
+.light-theme :global(.secondary-button),
+modal :global(.secondary-button):focus {
```

Comment:
Is `modal` supposed to be an element instead of a class or pseudo-class?

Reply:
good catch, the was a artifact from moving these from another style sheet